src/vs/workbench/browser/parts/editor/editorGroupView.ts

```diff
@@ -1646,14 +1646,20 @@ export class EditorGroupView extends Themable implements IEditorGroupView {
 		}
 
 		// Handle inactive first
-		for (const { editor, replacement, options } of inactiveReplacements) {
+		for (const { editor, replacement, skipSaveDialog, options } of inactiveReplacements) {
 
 			// Open inactive editor
 			await this.doOpenEditor(replacement, options);
 
 			// Close replaced inactive editor unless they match
 			if (!editor.matches(replacement)) {
-				const closed = await this.doCloseEditorWithDirtyHandling(editor, { preserveFocus: true });
+				let closed = false;
+				if (skipSaveDialog) {
+					this.doCloseEditor(editor);
```

Comment:
When you call `doCloseEditor` you need to set the `focusNext` param accordingly. Check how that param is set from `doCloseEditorWithDirtyHandling` depending on the `preserveFocus` flag.

Reply:
Have respsected tha flag now.